modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/select_site_role.jspf

```diff
@@ -31,7 +31,7 @@ if (step == 1) {
 	String keywords = ParamUtil.getString(request, "keywords");
 
 	if (Validator.isNull(keywords)) {
-		groups = selUser.getGroups();
+		groups = GroupLocalServiceUtil.getUserGroups(selUser.getUserId(), true);
```

Comment:
`GroupLocalServiceUtil.getUserGroups` include all groups (sites, org, user groups, etc). We should include only the site groups.